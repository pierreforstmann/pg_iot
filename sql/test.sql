--
-- test.sql
--
create extension pg_iot;
--
create table t(x int);
insert into t values(1);
select * from t;
update t set x=2;
select * from t;
delete from t;
select * from t;
--
select pg_iot_set('public','t');
insert into t values(1);
select * from t;
delete from t;
select * from t;
update t set x=2;
select * from t;

