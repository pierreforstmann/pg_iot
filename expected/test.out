--
-- test.sql
--
create extension pg_iot;
--
create table t(x int);
insert into t values(1);
select * from t;
 x 
---
 1
(1 row)

update t set x=2;
select * from t;
 x 
---
 2
(1 row)

delete from t;
select * from t;
 x 
---
(0 rows)

--
select pg_iot_set('public','t');
 pg_iot_set 
------------
 t
(1 row)

insert into t values(1);
select * from t;
 x 
---
 1
(1 row)

delete from t;
ERROR:  pg_iot: UPDATE and DELETE operations are disabled on table "public.t"
select * from t;
 x 
---
 1
(1 row)

update t set x=2;
ERROR:  pg_iot: UPDATE and DELETE operations are disabled on table "public.t"
select * from t;
 x 
---
 1
(1 row)

